.github/scripts/diff-suppression-keys-with-docs.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# this script reports some false positives, but is helpful in comparing the keys listed
+# in suppressing-instrumentation.md with the keys listed in the source files
+
+set -e -u -o pipefail
+
+comm -3 \
+  <(
+    sed -n '/----------------------/,${p;/^$/q}' \
+        ../opentelemetry.io/content/en/docs/instrumentation/java/automatic/agent-config.md \
```

Comment:
Does it work with a relative path? Shouldn't that md file be downloaded first?

Reply:
ah, no, this relies on having that other repo locally at that location.

but I like the idea of downloading the `agent-config.md` file instead, will update